src/main/java/ch/securify/analysis/MayImplicitDataflow.java

```diff
@@ -29,7 +29,7 @@
     public MayImplicitDataflow(){}
 
     public MayImplicitDataflow(List<Instruction> decompiledInstructions) throws IOException, InterruptedException {
-        SOUFFLE_RULES = "smt_files/mayImplicit.dl";
+        DL_EXEC = "/bin/mayImplicit";
```

Comment:
Isn't this going to break outside Docker?

Reply:
Oh yes I didn't think of that!! What would you suggest in order to make it work outside the docker? I guess either we could provide an "install script" that builds the datalog executables outside of the docker, or check for an environment variable, in order to determine whether to use a binary or the interpreter.

Reply:
IMO providing an install script that does the setup is probably smarter, since it means the behaviour of Securify does not differ between the docker and running it outside the docker. What do you think?

Reply:
These sound interesting, but isn't it possible to just keep the paths relative as they currently are?

Reply:
Yes sure, the only thing is that the paths should relate between the Dockerfile and the call here. I just thought `/bin` should be suitable to place the binaries. Either way, it does not solve the problem of running it outside the docker. The binaries will have to be created such that the Java program can find them, either dynamically via some sort of search, or statically such as the solution presented above.

Reply:
Good point. In a way I feel that this compilation step belongs to the build system, so the whole gradle thing, rather than Docker. I'll look into it, that should not be too difficult, now that you've written the right command to compile the Soufflé files.